### ISSUE TITLE
FEATURE REQUEST: Add a flag and feedback to clear all existing local records

### ISSUE THREAD

**Issue by iamGBOX** [t-a-y-l-o-r/checkip#33]
## Detailed Description

It would be convenient to be able to use a flag on checkip to clear all existing records to check a set of addresses totally fresh, rather than running `checkip -f` or manually deleting the json records.

## Context

This would permit a complete clear of all existing records from within the tool, which could streamline new investigations.

## Possible Implementation

```
checkip -clear

Are you sure you want to clear ALL existing local records? (Y/N)

Local records cleared
```

## Your Environment

* Version used: Current 
* Operating System and version (desktop or mobile): macOS 11.4

**Comment by t-a-y-l-o-r** [comment]
I could see an argument for adding this in as a flag like:

`checkip --fresh -ip 8.8.8.8` would create a fresh record. Would behave similar to `checkip --force -ip 8.8.8.8` but would also be a destructive action.

could also see:
`checkip --no-record -ip 8.8.8.8` if you want to avoid adding this ip to the record if you know you scan it a lot.

Long term adding all this into the config is planned *eventually.* But currently have no timeline for that

**Comment by iamGBOX** [comment]
I think there's definite utility in both of those options, the, `checkip --fresh -ip 8.8.8.8` and the `checkip --no-record -ip 8.8.8.8`, but I would also find the option to totally clear all local records useful, especially since the security status of records continues to become less relevant as they age.

**Comment by t-a-y-l-o-r** [comment]
In practice are you asking for:

```sh
# note! no ip given
$ checkip --delete-cache
```

Which would delete all local records across the board. (i.e. the record and the report)

**Comment by iamGBOX** [comment]
Yes! That's more what I was thinking of, an option to delete all local records across the board.

EDIT: A colleague suggested that I could use an alias in my shell to emulate this behaviour for now, but having that natively would be helpful.

**Comment by t-a-y-l-o-r** [comment]
sounds good. Won't be in this current set of fixes. But could easily be on the shortlist of new features after that

**Comment by t-a-y-l-o-r** [comment]
local testing went well. Just need confirmation from @iamGBOX
